packages/react-instantsearch-hooks-dom/src/widgets/SearchBox.tsx

```diff
@@ -3,17 +3,17 @@ import { useSearchBox } from 'react-instantsearch-hooks';
 
 import { SearchBox as SearchBoxUiComponent } from '../ui/SearchBox';
 
+import type { Translatable } from '../types';
 import type { SearchBoxProps as SearchBoxUiComponentProps } from '../ui/SearchBox';
-import type { ChangeEvent } from 'react';
 import type { UseSearchBoxProps } from 'react-instantsearch-hooks';
 
 export type SearchBoxProps = Omit<
-  SearchBoxUiComponentProps,
+  Translatable<SearchBoxUiComponentProps>,
```

Comment:
I thought we weren't going to make the translations available top-level yet?

Reply:
Absolutely, I think I effed up a rebase. Addressed 423f4775.